### ISSUE TITLE
🐛 [BUG] - Missing space between verify channel and opt-in button

### ISSUE THREAD

**Issue by hrithik73** [ethereum-push-notification-service/push-dapp#1433]
### Expected Behaviour

There should be a margin between 'Verify Channel' and 'Otp-In' button

### Current Behaviour

There is no margin between the button
<img width="788" alt="Screenshot 2024-03-04 at 12 53 29 PM" src="https://github.com/ethereum-push-notification-service/push-dapp/assets/48889727/547695a2-7862-4cf7-b01c-805f9eed4556">

### Reproduction steps

```bash
1. Go to 'https://app.push.org/channels'
2. Scroll down to the channels which are not verified
```

### Dapp Env

Prod (app.push.org)

### Browsers

Chrome

**Comment by HarshRajat** [comment]
https://github.com/ethereum-push-notification-service/push-dapp/pull/1435

**Comment by hrithik73** [comment]
I verified in alpha and it was fixed , and now #1435 is merged i will be fixed i guess but i am not able to verify it in live. so closing this
